haiku-apps/wakeup/wakeup-1.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="An application that plays a sound after a given time"
+DESCRIPTION="WakeUp is a timer application that allows users \
+to set their own ringtone, giving them a more personalized \
+experience. This application could be used for many purposes, \
+like an alarm clock or a cooking timer, just to name a few."
+HOMEPAGE="https://github.com/HaikuArchives/WakeUp"
+COPYRIGHT="1999 Jonathan Villemure"
+LICENSE="MIT"
+REVISION="1"
+gitSrcRevision="116afedea73ba9779d3d641028b65fdb2dc19012"
+SOURCE_URI="https://github.com/HaikuArchives/WakeUp/archive/$gitSrcRevision.tar.gz"
+CHECKSUM_SHA256="28eb8504a2a436a117821aaa399b8c37f5363bd9fcb2ff814f6cb1b6264da311"
+SOURCE_DIR="WakeUp-$gitSrcRevision"
```

Comment:
haikuporter has a nice feature which allows to rename the tarball right after the download.
To use it, you just need to define SOURCE_FILENAME. The recommended place to put it is between CHECKSUM_SHA256 and SOURCE_DIR (as stated in [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)):
```
SOURCE_FILENAME="WakeUp-$gitSrcRevision.tar.gz"
```

Reply:
That is a really nice feature. A great way to change "116afedea73ba9779d3d641028b65fdb2dc19012.tar.gz" to something much nicer.